processor/cumulativetodeltaprocessor/config.go

```diff
@@ -24,15 +24,19 @@ import (
 type Config struct {
 	config.ProcessorSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// List of cumulative sum metrics to convert to delta
+	// List of cumulative metrics to convert to delta. Default: converts all cumulative metrics to delta.
 	Metrics []string `mapstructure:"metrics"`
+
+	// The total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
+	MaxStale time.Duration `mapstructure:"max_stale"`
```

Comment:
I am not sure about the use case here. Wondering, how important is it to expose as a config option? Less configuarability gives us simplicity.

Reply:
The use case is for the prometheus receiver. The scrape interval is configurable and so we wouldn't want metrics to go stale prior to the scraper completing. Allowing this value to be configurable means that users can configure the staleness with their scrape interval + scrape duration + fudge factor.

Reply:
Looks good. I suggest `MaxStaleness`.